### DISCUSSION TITLE
Keyboard shortcuts not working on Mac

### DISCUSSION THREAD

**Discussion by bradlorge** [vkbo/novelWriter#601] [Help] [answered]
Hi! Thank you novelWriter team for releasing this super interesting App!

I can't seem to get the keyboard shortcuts for adding tags etc to work on Mac. 

MacOS version: 10.15.7
novelWriter version 1.0.3

Any help would be very much appreciated.

**Comment by vkbo** [comment, marked as answer]
These are two-step shortcuts. You press `ctrl+k` first, then the key of the insert you want to make. For Mac, replace `ctrl` with `cmd`. It's [described in the documentation](https://novelwriter.readthedocs.io/en/stable/usage_interface.html#insert-shortcuts).

Edit: Updated link

**Reply by bradlorge** [reply]
Thank you! I swear I had been doing that but perhaps not because it's working now. Appreciate you providing support!

**Reply by vkbo** [reply]
Great! Remember to click the check mark in the right corner of my response to mark it as answered.
